andela/persephone-ah-frontend
# #166816057 Setup react router

#### What does this PR do?
Set up react-router

#### Description of Task to be completed?
> - Add react-router
> - Create a demo login page to show that the router works
 
#### How should this be manually tested?
> - clone this branch
> - run `npm install` and `npm run dev`
> - it should open the apps demo home page in your default browser
> - this page should contain links to login page
> - click on this link and you should view the login page, and notice the URL also changes

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#166816057](https://www.pivotaltracker.com/story/show/166816057)

#### Screenshots (if appropriate)
![react-router](https://user-images.githubusercontent.com/33371467/62122838-8a940e80-b2be-11e9-9161-fe99f365c6f3.gif)

#### Questions:
NA
